internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -547,11 +554,9 @@ func TestAccClusterAdvancedClusterConfig_replicationSpecsAnalyticsAutoScaling(t
 
 func TestAccClusterAdvancedClusterConfig_replicationSpecsAndShardUpdating(t *testing.T) {
 	var (
-		orgID            = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName      = acc.RandomProjectName()
-		clusterName      = acc.RandomClusterName() // No ProjectIDExecution to avoid cross-region limits because multi-region
-		numShards        = "1"
-		numShardsUpdated = "2"
+		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName = acc.RandomProjectName()
+		clusterName = acc.RandomClusterName() // No ProjectIDExecution to avoid cross-region limits because multi-region
```

Comment:
should this comment be on the line above? same for the other test

Reply:
thx, changed